modules/nw-dns-operatorloglevel.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+// * networking/dns-operator.adoc
+
+:_content-type: PROCEDURE
+[id="nw-dns-operatorloglevel_{context}"]
+= Setting CoreDNS Operator log level
+
+Cluster Administrators can configure `operatorLogLevel` using the `logrus` parameter to more quickly track down OpenShift DNS issues. The `logrus` parameter has seven logging levels: Trace, Debug, Info, Warning, Error, Fatal and Panic. After the logging level is set, log entries with that severity or anything above it will be logged. Valid values for operatorLogLevel are `Normal`, `Debug`, and `Trace`. `Trace` has the most detailed information.
```

Comment:
* Cluster Administrators -> Cluster administrators
* Suggest changing this intro sentence like you did in the above module. "... can configure the Operator log level using the ... Though this actually confuses me on what the "logrus" parameter is, because in the procedure below it looks like the parameter that is used is called `operatorLogLevel`
* Should the seven logging levels (Trace, Debug, etc. ) be marked up in backticks?
* Suggest marking up operatorLogLevel in the second to last sentence in backticks